### PR TITLE
fix(nil): ingress nil panic

### PR DESCRIPTION
**Pull request by cndoit18**
Signed-off-by: cndoit18 <cndoit18@outlook.com>

fixes: #715

### CODE REVIEW

File: manifests/samples/full_stack_v1beta1.yaml [hunk 1]
```diff
@@ -86,6 +86,12 @@ spec:
         controller: default
       tls:
         certificateRef: sample-public-certificate
+    portal:
+      ingress:
+        host: notary.harbor.domain
```
Comment (steven-zou) [inline manifests/samples/full_stack_v1beta1.yaml:91]:
```suggestion
        host: portal.harbor.domain
```

File: manifests/samples/standard_stack_v1beta1.yaml [hunk 2]
```diff
@@ -220,6 +220,12 @@ spec:
         controller: default
       tls:
         certificateRef: sample-public-certificate
+    portal:
+      ingress:
+        host: notary.harbor.domain
```
Comment (steven-zou) [inline manifests/samples/standard_stack_v1beta1.yaml:225]:
```suggestion
        host: portal.harbor.domain
```

**Review by steven-zou — CHANGES_REQUESTED** [review]
please do changes

**Review by HenryKamg — APPROVED** [review]
LGTM

**Review by bitsf — APPROVED** [review]

**Pull request merged**